docs/sample_schema_package/MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+include LICENSE README
+include mailman_schema/tests/*
```

Comment:
I'd expect this to be picked up without needing to list it in the manifest since it's no longer being excluded by the find_package() call. Is that not the case?

Reply:
Removed the line to include the tests. Tested the MANIFEST without it and it works.